$OpenBSD: patch-scan-engine_cc,v 1.12 2019/08/24 06:37:13 giovanni Exp $

pcap_filter changes: to deal with base libpcap which doesn't support sctp
(not currently needed as using internal libpcap)

Format changes for 64-bit time_t, should go upstream

Index: scan_engine.cc
--- scan_engine.cc.orig
+++ scan_engine.cc
@@ -2059,7 +2059,7 @@ void ultrascan_host_probe_update(UltraScanInfo *USI, H
     struct timeval tv;
 
     gettimeofday(&tv, NULL);
-    log_write(LOG_STDOUT, "%s called for machine %s state %s -> %s (trynum %d time: %ld)\n", __func__, hss->target->targetipstr(), readhoststate(hss->target->flags), readhoststate(newstate), probe->tryno, (long) TIMEVAL_SUBTRACT(tv, probe->sent));
+    log_write(LOG_STDOUT, "%s called for machine %s state %s -> %s (trynum %d time: %lld)\n", __func__, hss->target->targetipstr(), readhoststate(hss->target->flags), readhoststate(newstate), probe->tryno, (time_t) TIMEVAL_SUBTRACT(tv, probe->sent));
   }
 
   ultrascan_host_pspec_update(USI, hss, probe->pspec(), newstate);
